Subject: [patch] fix data types in stncasecmp calls
From: Robert Schwebel <r.schwebel@pengutronix.de>

ne_mesg.c:150: error: pointer targets in passing argument 1 of 'strncasecmp' differ in signedness

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 ne_mesg.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: totd-1.5/ne_mesg.c
===================================================================
--- totd-1.5.orig/ne_mesg.c
+++ totd-1.5/ne_mesg.c
@@ -147,7 +147,7 @@ int mesg_dname_cmp (u_char *msg, u_char 
 		if (*dname == EDNS0_ELT_BITLABEL) {
 			if (memcmp (dname_mesg + 1, dname + 1,  len))
 				return -1;
-		} else if (strncasecmp (dname_mesg + 1, dname + 1, len))
+		} else if (strncasecmp ((const char*)dname_mesg + 1, (const char*)dname + 1, len))
 				return -1;
 
 		dname += len + 1;
